addons/mass_mailing/views/themes_templates.xml

```diff
@@ -17,6 +17,23 @@
 
     <!-- Default Theme -->
     <template id="theme_default_template">
+        <style id="design-element">
+            a:not(.btn), a.btn-link {
+                color: #81709b;
+            }
+            a.btn-primary, a.btn-outline-primary, a.btn-fill-primary {
+                font-size: 14px;
+                color: #ffffff;
+                background-color: #81709b;
+                border-color: #81709b;
+            }
+            a.btn-secondary, a.btn-outline-secondary, a.btn-fill-secondary {
+                font-size: 14px;
+                color: #FFFFFF;
+                background-color: #72e1d0;
+                border-color: #72e1d0;
+            }
+        </style>
```

Comment:
As mentioned in the Enterprise branch, for the moment if you redefine #design-element, you need to define all the default selectors, and for each of them all their styles, using the defaults in `DEFAULT_CSS_OBJECT`. That will be addressed later but right now that's a limitation. Otherwise the design tab won't know which styles are currently applied.

Reply:
Also, could you describe your changes in your commit message please?

Reply:
@edi-odoo **PING**